mwzeval/metrics.py

```diff
@@ -293,11 +293,11 @@ def get_dst(input_data, reference_states, include_loocv_metrics=False, fuzzy_rat
     """
     DOMAINS = {"hotel", "train", "restaurant", "attraction", "taxi"}
 
-    def block_domains(input_states: dict, reference_states: dict, blocked_domains: set[str]) -> dict:
+    def block_domains(input_states: dict, reference_states: dict, included_domains: set[str]) -> dict:
         """Return new input and reference state dictionaries with the specified domains removed.
```

Comment:
Also, let's call this `filter_inputs_and_references` so that the intent of the function is clear from the definition. `block_domains` is rather colloquial and ambiguous.